web_widget_digitized_signature/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Web Digitized Signature
+=====================
```

Comment:
@eLBati this `==` lines have to have same length as the text contained between them